### ISSUE TITLE
WCAG Requester signupflow: Input field amount/number

### ISSUE THREAD

**Issue by jamalv** [teamforus/forus#2237]
In the application process in step 2, the self-made element to select the number of children has no name that describes what this element does and no role to tell auxiliary software what function this element has.

Auxiliary software cannot tell its user that it can enter the number of children.

The most robust solution is to use the native HTML elements such as input elements. Another possible solution is to use WAI-ARIA. 

![image](https://user-images.githubusercontent.com/10818702/111795729-c16dcc80-88c7-11eb-86bd-84474cac4889.png)
